apps/publisher/extensions/assets/default/asset.js

```diff
@@ -313,8 +313,9 @@ asset.renderer = function(ctx) {
                 navList.push('Lifecycle', 'btn-lifecycle', util.buildUrl('lifecycle') + '/' + id);
             }
         }
-        if (permissionAPI.hasActionPermissionforPath(path, 'write', ctx.session) && permissionAPI.hasAssetPagePermission(type,'update',user.tenantId,username)) {
-           navList.push('Version', 'btn-copy', util.buildUrl('copy') + '/' + id);
+        if (permissionAPI.hasActionPermissionforPath(path, 'write', ctx.session)
```

Comment:
Suggestion shall we change this if statement to:
var hasWritePermission = permissionAPI.hasActionPermissionforPath(path, 'write', ctx.session);
var hasUpdatePermission = permissionAPI.hasAssetPagePermission(type,'update',user.tenantId,username);

if ( hasWritePermission && hasUpdatePermission && page.assets.version ) {